src/hal/m5fire.h

```diff
@@ -0,0 +1,63 @@
+// clang-format off
+// upload_speed 921600
+// board m5stack-fire
+
+// EXPERIMENTAL VERSION - NOT TESTED ON M5 HARDWARE YET
+
+#ifndef _M5FIRE_H
+#define _M5FIRE_H
+
+#include <stdint.h>
+
+// #define HAS_LORA 1 // comment out if device shall not send data via LoRa or has no M5 RA01 LoRa module
+
+// Pins for LORA chip SPI interface, reset line and interrupt lines
+#define LORA_SCK  SCK
+#define LORA_CS   SS
+#define LORA_MISO MISO
+#define LORA_MOSI MOSI
+#define LORA_RST  GPIO_NUM_36
+#define LORA_IRQ  GPIO_NUM_26
+#define LORA_IO1  GPIO_NUM_34 // must be externally wired on PCB!
+#define LORA_IO2  LMIC_UNUSED_PIN
+
+
+// enable only if you want to store a local paxcount table on the device
+#define HAS_SDCARD  1      // this board has an SD-card-reader/writer
+// Pins for SD-card
+#define SDCARD_CS    GPIO_NUM_4
+#define SDCARD_MOSI  MOSI
+#define SDCARD_MISO  MISO
+#define SDCARD_SCLK  SCK
+
+// user defined sensors
+//#define HAS_SENSORS 1 // comment out if device has user defined sensors
+
+#define CFG_sx1276_radio 1 // select LoRa chip
+#define BOARD_HAS_PSRAM // use if board has external PSRAM
+#define DISABLE_BROWNOUT 1 // comment out if you want to keep brownout feature
+
+//#define HAS_DISPLAY 1
+#define HAS_TFT 1
```

Comment:
Dont know how you want to implement the feature.
I think you also need to specify the Controller (ILI9341)

Reply:
Yes, we need some way to specify the controller here. Maybe like GPS_SERIAL, look in generic.h